provisioning/miniprov/hare_mp/main.py

```diff
@@ -254,21 +255,202 @@ def test(args):
         exit(-1)
 
 
+def executecmds(cmd: List[str]) -> list:
```

Comment:
Let it be just `execute`. Cmds is misleading (it executes just one command only).